packages/cli/src/commands/login/index.ts

```diff
@@ -68,14 +71,24 @@ export default async function login(client: Client): Promise<number> {
     return result;
   }
 
-  const isNewLogin = !client.authConfig.token;
-
   // Save the user's authentication token to the configuration file.
   client.authConfig.token = result.token;
 
-  // If we have a new login, update `currentTeam`
-  if (isNewLogin) {
-    await updateCurrentTeamAfterLogin(client, output, result.teamId);
+  // Save the user's team (possibly `undefined`) to the global configuration file.
+  client.config.currentTeam = result.teamId;
```

Comment:
This handles both setting it to a new value _and_ unsetting the old value. We later `delete` this key entirely if it remains `undefined`

Reply:
Maybe this could be more readable by using an if/else?

```
if (result.teamId) {
  client.config.currentTeam = result.teamId;
} else {
  delete client.config.currentTeam;
}
```